ui/app/AppLayouts/Profile/views/PrivacyAndSecurityView.qml

```diff
@@ -37,7 +44,63 @@ SettingsContentBase {
                 }
             ]
             onClicked: root.setNewsRSSEnabledRequested(!statusNewsSwitch.checked)
+            enabled: root.thirdpartyServicesEnabled
+        }
+
+        // Divider
+        Rectangle {
+            Layout.preferredWidth: root.contentWidth
+            Layout.preferredHeight: 1
+            color: Theme.palette.baseColor2
         }
+
+        StatusListItem {
+            Layout.preferredWidth: root.contentWidth
+            title: qsTr("Third-party services")
+            subTitle: qsTr("Enable/disable all third-party services")
+            components: [
+                StatusSwitch {
+                    checked: root.thirdpartyServicesEnabled
```

Comment:
Already written in the global comments.. There's a `double-check` action needed, so that, there's a middle state a bit confusing